app/models/miq_cockpit_ws_worker/runner.rb

```diff
@@ -150,8 +150,9 @@ def cockpit_ws_run
     }
     Bundler.with_clean_env do
       stdin, stdout, stderr, wait_thr = Open3.popen3(env, *cockpit_ws.command(BINDING_ADDRESS), :unsetenv_others => true)
+      stdin.close
```

Comment:
Did you try 

```ruby
stdin, stdout, stderr, wait_thr = Bundler.with_clean_env do
  Open3.popen3(env, *cockpit_ws.command(BINDING_ADDRESS), :unsetenv_others => true)
end
stdin.close
```

We need `wait_thr`, `stdout`, `stderr` to be in the same lexical scope as the return statement at the end too.

Reply:
right, they should not be defined inside block but used outside.

Also, looking at https://www.rubydoc.info/stdlib/open3/Open3.popen3 , non-block form of ```Open3.popen3``` invocation required all streams to be closed explicitly.

Also, path to ```cockpit.conf``` points to not existing ```.../lib/config/cockpit``` instead of ```.../vmdb/config/cockpit```,   may be it is the root cause of reported issue